package.json

```diff
@@ -23,17 +23,20 @@
     "@pocket-tools/apollo-utils": "^2.2.0",
     "@sentry/node": "^6.17.5",
     "@sentry/tracing": "^6.17.4",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
     "apollo-datasource-rest": "^3.5.1",
+    "apollo-link-http": "^1.5.17",
     "apollo-server": "^3.4.1",
     "apollo-server-cache-redis": "^3.3.1",
     "apollo-server-express": "^3.4.1",
     "apollo-server-plugin-response-cache": "^3.3.0",
     "aws-xray-sdk-express": "^3.3.4",
-    "dataloader": "^2.0.0",
-    "graphql": "^16.3.0",
-    "graphql-tag": "^2.12.6",
     "cross-fetch": "3.1.5",
-    "express": "4.17.3"
+    "dataloader": "^2.0.0",
+    "express": "4.17.3",
+    "graphql": "^15.8.0",
```

Comment:
Had to downgrade the `graphql` package to be able to use `apollo-link-http`.

Reply:
Heads up that apollo-link is deprecated https://github.com/apollographql/apollo-link They recommend using apollo client and the packages are moved into there. I don't think you need to update it now in this pr, but something to keep in mind for future work.

Reply:
looks like this is the new way? https://www.apollographql.com/docs/react/api/link/apollo-link-http/

would be good to update our other repos, too.

Reply:
Fixed in this PR - as far as I know, we have this setup in the FF Android Proxy repo as well - it should be a quick fix to update it there.